src/main.cpp

```diff
@@ -1821,7 +1510,7 @@ void setup() {
         .type = kDouble,
         .section = sBDSection,
         .position = 20,
-        .show = [] { return true && FEATURE_BREWDETECTION == 1 && useBDPID; },
```

Comment:
You'll probably want to remove all of these vars as their show condition is only true in case the BD feature is enabled and the flag useBDPID is true. That will never be the case when you remove the feature